.codecov.yml

```diff
@@ -0,0 +1 @@
+comment: off
```

Comment:
I think this will fail the rat check. Needs a header like in .travis.yml. Otherwise looks good.
+1